ecl/eclagent/eclagent.ipp

```diff
@@ -1050,9 +1058,17 @@ class EclGraph : public CInterface
         {
             container = _container;
         }
-
+        virtual unsigned getWorkflowId() const override
```

Comment:
similar to prev. comments. There's a mixture of methods (now) some missing both virtual and override.
It would clarify the code if interface implementation methods were grouped together and made consistent.